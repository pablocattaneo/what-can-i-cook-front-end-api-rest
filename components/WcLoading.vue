<template>
  <div id="wc-loading" class="text-center">
    <b-spinner
      variant="primary"
      class="wc-loading-spinner"
      style="width: 3rem; height: 3rem;"
    />
  </div>
</template>
