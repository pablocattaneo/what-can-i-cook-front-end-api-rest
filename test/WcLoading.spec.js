import { shallowMount, createLocalVue } from '@vue/test-utils'
import WcLoading from '@/components/WcLoading'
import { SpinnerPlugin } from 'bootstrap-vue'
const localVue = createLocalVue()

let wrapper
beforeEach(() => {
  localVue.use(SpinnerPlugin)
  wrapper = shallowMount(WcLoading, { localVue })
})

test('It verify if non-intentional changes were made.', () => {
  expect(wrapper.element).toMatchSnapshot()
})

test('should have wc-loading id', () => {
  expect(wrapper.attributes('id')).toBe('wc-loading')
})
