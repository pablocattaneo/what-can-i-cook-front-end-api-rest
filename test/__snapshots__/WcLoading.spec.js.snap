// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="text-center"
  id="wc-loading"
>
  <b-spinner-stub
    class="wc-loading-spinner"
    role="status"
    style="width: 3rem; height: 3rem;"
    tag="span"
    type="border"
    variant="primary"
  />
</div>
`;
